Utils/fetchUtils.js

```diff
@@ -12,4 +12,14 @@ export const fetchArticles = () => {
     .catch((err) => {
       console.log(err);
     });
-};
\ No newline at end of file
+};
+
+export const fetchArticleById = (id) => {
+  return NCnewsAPI.get(`/api/articles/${id}`)
+      .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
This shouldn't be needed your API should be responding with an error if something goes wrong